src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -243,14 +243,23 @@ public void DirectoryLongerThanMaxLongPath_ThrowsPathTooLongException()
 
         [ConditionalFact(nameof(LongPathsAreNotBlocked), nameof(UsingNewNormalization))]
         [ActiveIssue(20117, TargetFrameworkMonikers.Uap)]
-        [PlatformSpecific(TestPlatforms.Windows)]  // long directory path with extended syntax throws DirectoryNotFoundException
-        public void DirectoryLongerThanMaxLongPathWithExtendedSyntax_ThrowsDirectoryNotFoundException()
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void DirectoryLongerThanMaxLongPathWithExtendedSyntax_ThrowsException()
```

Comment:
@JeremyKuhne So, I was wondering how I got a bunch of the tests I already had passing to fail again. It seems that this following code generates very different paths after I renamed this test method. That seems like a pretty scary side effect. Do you know what the best course of action here would be?